src/test/ui/unsized-locals/unsized-parameters.rs

```diff
@@ -1,6 +1,7 @@
 // run-pass
 
-#![feature(unsized_locals)]
+#![allow(incomplete_features)]
+#![feature(unsized_locals, unsized_fn_params)]
```

Comment:
Why does this need `unsized_locals`?

Reply:
Ohh this one without `unsized_locals` gives ...

```
error[E0161]: cannot move a value of type [i32]: the size of [i32] cannot be statically determined
 --> src/test/ui/unsized-locals/unsized-parameters.rs:7:16
  |
7 | pub fn f2((_x, _y): (i32, [i32])) {}
  |                ^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0161`.
```

Reply:
That makes sense, actually. The way patterns work, this desugars to some MIR-level moving. So `unsized_fn_params` should only capture the case `x: Unsized`.